src/components/common/Navbars/NavBar.js

```diff
@@ -70,7 +70,7 @@ function NavBar(props) {
       style={{
         backgroundColor: bgColor,
         minHeight: '98px',
-        lineHeight: '0',
+        lineHeight: '0px',
```

Comment:
something to consider moving forward -- that is outside the scope of this ticket -- is that we should open an Issue to clean up the inline styles throughout the app.